### ISSUE TITLE
It's not clear that you can scroll to view long gallery image captions. 

### ISSUE THREAD

**Issue by kjellr** [WordPress/gutenberg#7963]
When a gallery image contains a long caption, users can scroll down to view the full caption text:

![gallery-long-captions](https://user-images.githubusercontent.com/1202812/42709056-2ea87004-86ad-11e8-8496-ef233aaee0f7.gif)

Unfortunately, if a users device isn't configured to show scrollbars (mobile devices, most Macs), there's no indication that scrolling is an option there. Instead, it just looks like text is missing. 

One possible idea: Cutting off the text slightly to imply that there's more to read:

![screen shot 2018-07-13 at 2 58 00 pm](https://user-images.githubusercontent.com/1202812/42709567-f2cb04aa-86ae-11e8-9382-a0acda65dbc3.png)

**Comment by kjellr** [comment]
I think needs design feedback is appropriate. 👍

**Comment by mapk** [comment]
Some quick feedback:

While the cutoff proposal for captions provides a solution for longer ones, it seems to make the shorter captions more difficult to read. Those would need to be scrolled too.

Maybe it's just the position of the cutoff. If we allow longer captions to move up in the Image and show the third line cut off, that might work better. This way shorter captions can still be completely visible and longer captions that need scrolling are also indicated.

![Screen Shot 2020-05-12 at 2 38 39 PM](https://user-images.githubusercontent.com/617986/81748245-5d37dc00-945e-11ea-8713-97dda063c480.png)

**Comment by paaljoachim** [comment]
At present time I think Mark's solution might be best. As it gives a hint that there is more text. Either the user thinks it might be a bug or will understand that they can scroll to view the rest of the caption.

**Comment by MDWolinski** [comment]
I agree with @mapk and @paaljoachim on the solution.  I do like @kjellr solution mockup with the slight gradient over the text to show it's contained within the image, however.

**Comment by joyously** [comment]
I think the best solution is to put the text below the image instead of on top of it. After all, it's an image gallery, and with long text, you can't even see the image.

**Comment by paaljoachim** [comment]
Btw here is an older issue which brings up gallery image captions:
https://github.com/WordPress/gutenberg/issues/8030

**Comment by paaljoachim** [comment]
Linking this to the PR.
Gallery: Refactor as wrapper around core image blocks
https://github.com/WordPress/gutenberg/pull/25940

**Comment by glendaviesnz** [comment]
This is more broken on the the new gallery format, so probably should be sorted before the new gallery experimental flag is removed:
![long-caption](https://user-images.githubusercontent.com/3629020/131269197-ed33122f-6ce5-40ce-b043-ece5426bd3a7.gif)

**Comment by glendaviesnz** [comment]
#34402 didn't really fix this so reopening

**Comment by mtias** [comment]
Outside of offering other styles, we are not really planning anything else here.
